cmake/Platform/Common/GCC/Configurations_gcc.cmake

```diff
@@ -73,6 +74,15 @@ ly_append_configurations_options(
         -Wno-switch
         -Wno-uninitialized
         -Wno-unused-result
+        -Wno-dangling-pointer
+        -Wno-error=unused-but-set-variable
+        -Wno-error=unused-variable
+        -Wno-error=subobject-linkage
+        -Wno-error=pragmas
```

Comment:
This shouldnt be needed with that warning disable statement above

Reply:
Removed -Wno-error=pragmas